docs/source/operations/getting-started.rst

```diff
@@ -0,0 +1,354 @@
+
+   brew install k3d
+   brew install kubectl
+   brew install skaffold
+   brew install helm
+   brew install gsed # Needed for k3s-create.sh
+
+
+First time configuration
+========================
+
+1. Execute the script :download:`k3-create.sh<./getting-started/k3-create.sh>`. This script deletes the existing cluster, recreates a new one and applies a patch for SSL.
+
+   1. Download :download:`k3-create.sh<./getting-started/k3-create.sh>`.
+   2. Make the script executable.
+
+      .. code-block:: bash
+
+         chmod +x ./k3-create.sh
+
+   3. Run the script
+
+      .. code-block:: bash
+
+         ./k3-create.sh
+
+   .. tip::
+      This script can be used to reset your development environment.
+
+2. Add the following line to ``/etc/hosts`` to allow the communication between your local cluster and the host (your machine):
+
+   .. code-block:: text
+
+      127.0.0.1 orchestrator.org-1.com orchestrator.org-2.com substra-frontend.org-1.com substra-frontend.org-2.com substra-backend.org-1.com substra-backend.org-2.com
+
+3. Add the helm repositories
+
+   .. code-block:: bash
+
+      helm repo add bitnami https://charts.bitnami.com/bitnami
+      helm repo add stable https://charts.helm.sh/stable
+      helm repo add twuni https://helm.twun.io
+      helm repo add jetstack https://charts.jetstack.io
+
+4. Clone the Substra components repositories
+
+   * `substra <https://github.com/substra/substra>`_
+
+     .. code-block:: bash
+
+      git clone https://github.com/Substra/substra.git
+
+   * `orchestrator <https://github.com/substra/orchestrator>`_
+
+     .. code-block:: bash
+
+      git clone https://github.com/Substra/orchestrator.git
+
+   * `substra-backend <https://github.com/substra/substra-backend>`_
+
+     .. code-block:: bash
+
+      git clone https://github.com/Substra/substra-backend.git
+
+   * `substra-frontend <https://github.com/substra/substra-frontend>`_
+
+     .. code-block:: bash
+
+      git clone https://github.com/Substra/substra-frontend.git
+
+
+5. Install substra in editable mode
+
+   .. code-block:: bash
+
+      cd substra
+      pip install -e .
+
+6. Install frontend dependencies
+
+   .. code-block:: bash
+
+      cd substra-frontend
+      npm install --dev
+
+Launching
+=========
+
+* Deploy the orchestrator
+
+  .. code-block:: bash
+
+   cd orchestrator
+   skaffold run
+
+.. _Deploy the backend:
+
+* Deploy the backend
+
+  .. code-block:: bash
+
+   cd substra-backend
+   skaffold run
+
+  .. caution::
+     On arm64 architecture (e.g. Apple silicon chips M1 & M2), you need to add the profiles ``dev``and ``arm64``.
+
+     .. code-block:: bash
+
+      skaffold run -p dev,arm64
+
+.. tip::
+   When re-launching the orchestrator and the backend, you can speed up the processing by avoiding the update of the chart dependencies using the profile ``nodeps``.
+
+   .. code-block:: bash
+
+      skaffold run -p nodeps
+
+* Deploy the frontend. You can use two methods (described below)
```

Comment:
You can also use skaffold run. Feel free to ignore my comment if you intentionally omitted it.

Reply:
I am not sure to understand this comment - this block is about speeding up the relaunch of orchestrator and backend by not updating chart dependencies.

Reply:
The line I highlighted is

> Deploy the frontend. You can use two methods (described below)

This is independent from orchestrator/backend relaunch?

Reply:
Ok that makes sense if it's the method that is documented :+1: 

Maybe it could be documented on the substra-frontend repo that the frontend can be started with `skaffold run` (that's what I use). cc @HamdyD. But it's out of scope for this PR.